compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmMethodGen.java

```diff
@@ -1207,7 +1207,8 @@ private static String getFrameClassName(String pkgName, String funcName, BType a
      * @return cleaned name
      */
     static String cleanupTypeName(String name) {
-
+        name = name.replace("/", "_");
+        name = name.replace(".", "_");
```

Comment:
@Kishanthan, this PR introduces some changes to support object/record intersection types, where for something like

```ballerina
import ballerina/http;

public function main() {
    http:Versioning & readonly v = {};
}
```

the name of the type of `v` has a name like `(ballerina/http:1.0.0:Versioning & readonly)`, with `/` and `.` in the name, which results in errors. So I've replaced them with `_` too.

I also had to call this method when [calculating the type field name](https://github.com/ballerina-platform/ballerina-lang/pull/23795/files#diff-49b683e4ffeab2be6da4ab1331bc3087R1455) (org.wso2.ballerinalang.compiler.bir.codegen.JvmTypeGen#getTypeFieldName), since that too would have these characters.

Do you see any issue with this change?